core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -174,6 +174,13 @@ private TableProperties() {}
   public static final String PARQUET_BLOOM_FILTER_COLUMN_ENABLED_PREFIX =
       "write.parquet.bloom-filter-enabled.column.";
 
+  public static final String PARQUET_COLUMN_STATS_ENABLED_PREFIX =
+      "write.parquet.stats-enabled.column.";
+
+  public static final String DEFAULT_PARQUET_COLUMN_STATS_ENABLED =
+      PARQUET_COLUMN_STATS_ENABLED_PREFIX + "default";
+  public static final boolean DEFAULT_PARQUET_COLUMN_STATS_ENABLED_DEFAULT = true;
+
```

Comment:
Do we need to document the new configs?

Reply:
Yes, it needs document for these new table properties.

Reply:
`configuration.md` could be updated to document the new configs

Reply:
will do